src/components/VerticalNavigation/VerticalNavigationItem.js

```diff
@@ -0,0 +1,215 @@
+            mobileItem && depth === 'primary' && showMobileTertiary,
+          // I don't know, that's just how this stuff was in patternfly-ng...
+        })}
+        onMouseEnter={() => {
+          handleHover[depth]('TODO args here');
+        }}
+        onMouseLeave={() => {
+          handleBlur[depth]('TODO args here');
+        }}
+      >
+        <a onClick={() => { handleClick[depth]('TODO args here') }}> {/* TODO handleClick takes all 3 items? figure that out... */}
+          {depth === 'primary' &&
+            iconStyleClass && (
+              <OverlayTrigger
+                placement="bottom"
+                overlay={navCollapsed ? <Tooltip>{title}</Tooltip> : null}
+              >
+                <span
+                  className={cx(iconStyleClass, { hidden: hiddenIcons })}
+                  title={title}
+                />
+              </OverlayTrigger>
+            )}
+          <span className="list-group-item-value">{title}</span>
+          {this.renderBadges(badges)}
+        </a>
+        {children &&
+          children.length > 0 && (
+            <div className="nav-pf-secondary-nav"> {/* TODO should this class sometimes say tertiary? */}
+              <div className="nav-item-pf-header">
+                <a
+                  className={cx(`${nextDepth}-collapse-toggle-pf`, {
+                    collapsed: nextDepthCollapsed,
+                  })}
+                  onClick={() => {
+                    collapseNextNav('TODO args here');
+                  }} // TODO what is the item arg in these calls?  some subset of props? pass what we need from it instead.
+                />
+                <span>{title}</span>
+              </div>
+              <ListGroup>
+                {VerticalNavigation.renderChildren(this.props)}
+              </ListGroup>
+            </div>
+          )}
+      </ListGroupItem>
+    );
+  }
+}
+
+VerticalNavigationItem.propTypes = {
+  title: PropTypes.string.isRequired,
+  trackActiveState: PropTypes.bool,
+  trackHoverState: PropTypes.bool,
+  mobileItem: PropTypes.bool,
+  iconStyleClass: PropTypes.string,
+  badges: PropTypes.shape({
+    badgeClass: PropTypes.string,
+    tooltip: PropTypes.string,
+    count: PropTypes.number,
+    iconStyleClass: PropTypes.string,
+  }),
+  showMobileSecondary: PropTypes.bool,
+  showMobileTertiary: PropTypes.bool,
+  navCollapsed: PropTypes.bool,
+  secondaryCollapsed: PropTypes.bool,
+  tertiaryCollapsed: PropTypes.bool,
+  handlePrimaryHover: PropTypes.func,
+  handleSecondaryHover: PropTypes.func,
+  handleTertiaryHover: PropTypes.func,
+  handlePrimaryBlur: PropTypes.func,
+  handleSecondaryBlur: PropTypes.func,
+  handleTertiaryBlur: PropTypes.func,
+  handlePrimaryClick: PropTypes.func,
+  handleSecondaryClick: PropTypes.func,
+  handleTertiaryClick: PropTypes.func,
+  children: PropTypes.node,
+};
+
+VerticalNavigationItem.defaultProps = {
+  trackActiveState: true,
+  trackHoverState: true,
```

Comment:
why are the defaults true here? Aren't these indicating the item is active or hovered on? Only one item at most should ever be active, same as hover.